### ISSUE TITLE
[ContentFilter] Don't show "You are serving a blocked site" notification to users who can't delete blocked site

### ISSUE THREAD

**Issue by defder-su** [ZeroNetX/ZeroNet-Plugins#5]
In the current implementation, the notification just advertises blocked content.

**Comment by slrslr** [comment]
I do not understand what is the problem, since i was always able to remove this notification from ZeroHello page by clicking the X at the end of it and continue hosting blocked site. I think that it is quite good like it is. Notification is important so user does not accidentally seed unwanted site.

When visiting blocked site, user is not bothered anyhow. And once he delete such site, he will be warned on re-visit - that is good.

**Comment by defder-su** [comment]
I'm talking about user who doesn't have permissions to delete blocked sites, in public gateways.

**Comment by slrslr** [comment]
I do not think You have to delete the site in order to get rid of yours mentioned notification, because i have mentioned: " i was always able to remove this notification from ZeroHello page by clicking the X at the end of it" and i have just tried to do it at [this public proxy](https://zerolink.ml/1HELLoE3sFD9569CLCbHEAVqvqV7U2Ri9d) (gateway?), reloaded page and it no longer appeared.
